### ISSUE TITLE
Activate image preview in GtkFileChooser

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#72]
```
Since GNOME 2.22 (more or less) GtkFileChooser offers the possibilty to
display a thumbnail of the selected image.
The application must ask for it.
Firefox uses it since 3.0 for instance.

It would be practical if KeepNote could also make use of it for Insert
image menu entry
```

Original issue reported on code.google.com by `thibaut....@gmail.com` on 19 Jun 2009 at 1:53

**Comment by GoogleCodeExporter** [comment]
```
see
http://library.gnome.org/devel/gtk/unstable/GtkFileChooser.html#gtk-file-chooser
-set-preview-widget
```

Original comment by `thibaut....@gmail.com` on 23 Jun 2009 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
Thanks for this info.  Image previews will be available in KeepNote 0.5.3.
```

Original comment by `matt.rasmus@gmail.com` on 7 Jul 2009 at 6:06
- Changed state: **Started**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This feature is now available in KeepNote 0.5.3
```

Original comment by `matt.rasmus@gmail.com` on 20 Jul 2009 at 4:07
- Changed state: **Done**
